kibble.json

```diff
@@ -10,7 +10,7 @@
     "fr": "fr_FR",
     "de": "de_DE",
     "es": "es_ES",
-    "es_MX": "es_MX",
+    "es-mx": "es-mx",
```

Comment:
Not sure how I feel about this being the only value that's hyphenated. Possibly live with it if the aim is to standardise over time.

Reply:
Yeah I just wasn't sure if I should do them all at once because every site that updates core after this will need the hyphens changing on their local lang filenames.

Reply:
Why can't it be `"es-mx": "es_MX",`?

Reply:
I'm sure that didn't work when I tried it out before but it does work so I'll change it.